packages/framework-core/src/services/graphql/graphql-mutation-generator.ts

```diff
@@ -9,8 +9,11 @@ export class GraphQLMutationGenerator {
     private readonly mutationResolver: ResolverBuilder
   ) {}
 
-  public generate(): GraphQLObjectType {
+  public generate(): GraphQLObjectType | undefined {
```

Comment:
Why do we have to return `undefined`?

Reply:
If we return a GraphQLObjectType with name 'Mutation' with empty fields, GraphQL library throws a validation error. By sending undefined to the GraphQLGenerator it creates the schema without mutations and without triggering a validation error.